packages/interactivity-router/src/index.ts

```diff
@@ -336,7 +339,7 @@ export const { state, actions } = store< Store >( 'core/router', {
 					?.clientNavigationDisabled
 			) {
 				try {
```

Comment:
Shouldn't this try-catch now wrap `regionsToVdom`, which is the one that can throw due to the `error` listener of the `link` elements?

Reply:
Right, I'll change this!

By the way, how do you think we should handle these errors inside `fetchPage`? I guess we would have to store an error response somehow so `navigate` knows that it should force a refresh (at least if we want to keep this logic). 🤔

Reply:
Previously, [we were saving `false` in the cache](https://github.com/woocommerce/gutenberg/blob/20c77f5e3510053e2a20e3258288fceb7b0c7521/packages/interactivity-router/src/index.ts#L78). Afterward, if we encountered a `false`, we would [force a refresh](https://github.com/woocommerce/gutenberg/blob/20c77f5e3510053e2a20e3258288fceb7b0c7521/packages/interactivity-router/src/index.ts#L159-L166). 

We can still use that pattern if that's not already in place (I haven't checked).

Reply:
Done at https://github.com/woocommerce/gutenberg/pull/10/commits/3201fafec1793381e9912df05a86412ad9a6c65e.